### ISSUE TITLE
Add PR Workflow for openebs-k8s-provisioner 

### ISSUE THREAD

**Issue by satyamz** [openebs/openebs#408]
FEATURE REQUEST

**What happened**:
Need PR workflow for contributing to openebs provisioner.
**What you expected to happen**:
- Create new branch on `openebs/external-storage`
- Raise PR to the `kubernetes-incubator/external-storage`
- Once PR is merged in  `kubernetes-incubator/external-storage`, sync the  `openebs/external-storage` master with `kubernetes-incubator/external-storage` master

By following above steps we can refrain from making mistakes on master branch

**Comment by satyamz** [comment]
Closing this in favour of #430
